tests/rptest/tests/compatibility/kafka_streams_test.py

```diff
@@ -98,11 +99,11 @@ class KafkaStreamsProdConsBase(KafkaStreamsTest):
     # The producer should be an extension to KafProducer
     PRODUCER = None
 
-    def __init__(self, test_context, proxy_conf, enable_sr):
+    def __init__(self, test_context, proxy_conf, registry_conf):
```

Comment:
I don't recall `schema_registry` every being shorted to `registry`, I don't think it's likely there will be much confusion, but I'm not sure the longer name is a problem either.

Reply:
I'll use `schema_registry_config` because it is more descriptive.